tests/countries/test_cape_verde.py

```diff
@@ -0,0 +1,127 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.countries import CapeVerde, CV, CAV
+from tests.common import CommonCountryTests
+
+
+class TestCapeVerde(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(CapeVerde, years=range(1973, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(CapeVerde, CV, CAV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(CapeVerde(years=1972))
+
+    def test_democracy_day(self):
+        name = "Dia da Liberdade e Democracia"
+        self.assertHolidayName(name, (f"{year}-01-13" for year in range(1991, 2050)))
+        self.assertNoHolidayName(name, CapeVerde(years=(1973, 1990)))
+
+    def test_independence_day(self):
+        name = "Dia da Independência Nacional"
+        self.assertHolidayName(name, (f"{year}-07-05" for year in range(1975, 2050)))
+        self.assertNoHolidayName(name, CapeVerde(years=(1973, 1974)))
```

Comment:
```suggestion
        self.assertNoHolidayName(name, range(1973, 1975))
```
Actually, if we change start_year to 1976, this line will be redundant.